rootfs/bin/boot

```diff
@@ -61,29 +57,10 @@ etcd_safe_mkdir /deis/platform
 etcd_safe_mkdir /deis/scheduler
 etcd_safe_mkdir /deis/services
 
-# HACK: set up keys for database
-etcd_safe_mkdir /deis/database
-ETCD_PATH=/deis/database etcd_set_default engine postgresql_psycopg2
-ETCD_PATH=/deis/database etcd_set_default name deis
-ETCD_PATH=/deis/database etcd_set_default user deis
-ETCD_PATH=/deis/database etcd_set_default password changeme123
-ETCD_PATH=/deis/database etcd_set_default host "${DEIS_DATABASE_SERVICE_HOST:-127.0.0.1}"
```

Comment:
The test database that `make kube-create-database` installs exports these environment variables. We don't want to use it long-term, but I think these are the appropriate env vars to expect.